rasa/core/utils.py

```diff
@@ -40,14 +40,18 @@ def configure_file_logging(
     """
     if not log_file:
         return
-
-    formatter = logging.Formatter("%(asctime)s [%(levelname)-5.5s]  %(message)s")
-    file_handler = logging.FileHandler(
-        log_file, encoding=rasa.shared.utils.io.DEFAULT_ENCODING
-    )
-    file_handler.setLevel(logger_obj.level)
-    file_handler.setFormatter(formatter)
-    logger_obj.addHandler(file_handler)
+    if log_file == "/dev/log":
```

Comment:
Is there a way we can confirm this is a syslog file? Relying on the path seems a bit brittle, especially cross-platform.

Reply:
Thanks for the review
I just changed this behavior by adding a new option to the command line. This way, it will still be a syslog file.
Thank you for the suggestion.